dev/reference/sql/statements/add-column.md

```diff
@@ -0,0 +1,82 @@
+---
+title: ADD COLUMN | TiDB SQL Statement Reference 
+summary: An overview of the usage of ADD COLUMN for the TiDB database.
+category: reference
+---
+
+# ADD COLUMN 
+
+The `ALTER TABLE.. ADD COLUMN` statement adds a column to an existing table. This operation is online in TiDB, which means that neither reads or writes to the table are blocked by adding a column.
+
+## Synopsis
+
+**AlterTableStmt:**
+
+![AlterTableStmt](/media/sqlgram/AlterTableStmt.png)
+
+**AlterTableSpec:**
+
+![AlterTableSpec](/media/sqlgram/AlterTableSpec.png)
+
+**ColumnKeywordOpt:**
+
+![ColumnKeywordOpt](/media/sqlgram/ColumnKeywordOpt.png)
+
+**ColumnDef:**
+
+![ColumnDef](/media/sqlgram/ColumnDef.png)
+
+**ColumnPosition:**
+
+![ColumnPosition](/media/sqlgram/ColumnPosition.png)
+
+## Examples
+
+```sql
+mysql> CREATE TABLE t1 (id INT NOT NULL PRIMARY KEY auto_increment);
+Query OK, 0 rows affected (0.11 sec)
+
+mysql> INSERT INTO t1 VALUES (NULL);
+Query OK, 1 row affected (0.02 sec)
+
+mysql> SELECT * FROM t1;
++----+
+| id |
++----+
+|  1 |
++----+
+1 row in set (0.00 sec)
+
+mysql> ALTER TABLE t1 ADD COLUMN c1 INT NOT NULL;
+Query OK, 0 rows affected (0.28 sec)
+
+mysql> SELECT * FROM t1;
++----+----+
+| id | c1 |
++----+----+
+|  1 |  0 |
++----+----+
+1 row in set (0.00 sec)
+
+mysql> ALTER TABLE t1 ADD c2 INT NOT NULL AFTER c1;
+Query OK, 0 rows affected (0.28 sec)
+
+mysql> SELECT * FROM t1;
++----+----+----+
+| id | c1 | c2 |
++----+----+----+
+|  1 |  0 |  0 |
++----+----+----+
+1 row in set (0.00 sec)
+```
+
+## MySQL compatibility
+
+* Adding multiple columns at the same time is currently not supported.
+* Adding a new column and setting it to the `PRIMARY KEY` is not supported.
+* Adding a new column and setting it to `AUTO_INCREMENT` is not supported.
+
+## See also
+
+* [ADD INDEX](add-index.md)
```

Comment:
This link will be a broken link at the PingCAP website. To avoid this issue, please use the link path rule that @YiniXu9506 told us. And please also update similar links involved in this PR.

Reply:
Thanks, fixed!